deploy/deployment.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     spec:
       serviceAccountName: calico-host-endpoint-controller
       containers:
-      - image: quay.io/kinvolk/calico-endpoint-controller:v0.0.2
+      - image: quay.io/kinvolk/calico-endpoint-controller:v0.0.3
```

Comment:
Please use the new location.
Either you follow the readme to publish it there (you need arm64 chroot with qemu-aarch64-static or maybe a ARM64 node on Packet to build the image) or I can do it.

Reply:
Moved to new location. Image isn't pushed yet though.